rts/lra/client/src/main/java/io/narayana/lra/client/NarayanaLRAClient.java

```diff
@@ -320,13 +321,6 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
             }
 
             lra = URI.create(response.getHeaderString(HttpHeaders.LOCATION));
-
-            if (lra == null) {
-                LRALogger.i18NLogger.error_nullLraOnCreation(response.toString());
```

Comment:
Please with this remove the method `error_nullLraOnCreation` from the `i18NLogger` as well.